bin/node/cli/tests/running_the_node_and_interrupt.rs

```diff
@@ -1,22 +1,55 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2020 Parity Technologies (UK) Ltd.
```

Didn't know that you can time travel :D